Tests/SentryTests/Protocol/SentrySdkInfoTests.swift

```diff
@@ -32,6 +33,31 @@ class SentrySdkInfoTests: XCTestCase {
             XCTFail("Serialization of SdkInfo doesn't contain sdk")
         }
     }
+
+    func testSPM_packageInfo() throws {
+        let version = "5.2.0"
+        let actual = SentrySdkInfo(name: sdkName, andVersion: version)
+        Dynamic(actual).packageManager = 0
+        let serialization = actual.serialize()
+
+        if let sdkInfo = serialization["sdk"] as? [String: Any] {
+            XCTAssertEqual(3, sdkInfo.count)
+
+            let packageInfo = try XCTUnwrap(sdkInfo["packages"] as? [String: Any])
+            XCTAssertEqual(packageInfo["name"] as? String, "spm:getsentry/\(sdkName)")
+            XCTAssertEqual(packageInfo["version"] as? String, version)
+        } else {
+            XCTFail("Serialization of SdkInfo doesn't contain sdk")
+        }
+    }
+
+    func testPackageNames () {
+        let actual = SentrySdkInfo(name: sdkName, andVersion: "")
+        XCTAssertEqual(Dynamic(actual).getPackageName(0), "spm:getsentry/%@")
+        XCTAssertEqual(Dynamic(actual).getPackageName(1), "cocoapods:getsentry/%@")
+        XCTAssertEqual(Dynamic(actual).getPackageName(2), "carthage:getsentry/%@")
+        XCTAssertNil(Dynamic(actual).getPackageName(3).asString)
```

Comment:
`m`: That test doesn't validate that `"cocoapods:getsentry/%@"` and `"carthage:getsentry/%@"` are proper format strings. Instead of asserting their values, it would be better to test the format strings like `testSPM_packageInfo` does.

Reply:
This test ensures that the format string don't change, the test before this one ensures that we use the format in the correct way.

Reply:
Yes, but if I now change `carthage:getsentry/%@` both here in the tests and in `SentrySDKInfo` to `carthage%@:getsentry/%@`, the test are green, but it should crash.

Reply:
If you change the implementation and the test, of course it would be green, same would happen for your suggestion.

But just to end this discussion Im gonna do it. This one is pretty much ctrl+c, ctrl+v.